kubernetes/keycloak/values-dev.yaml

```diff
@@ -61,10 +61,10 @@ keycloak:
   extraContainers: |
 
   ## Custom script that is run before Keycloak is started.
-  preStartScript: |
```

Comment:
Is there a reason `values-dev.yaml` doesn't have `script.sh: |` but `values.yaml` does?